### PR TITLE
Integrate Headway App

### PR DESCRIPTION
**Pull request by RaiVaibhav**

### CODE REVIEW

File: app/utils/trigger-headwayapp.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// jshint ignore: start
+import ENV from 'irene/config/environment';
+
+const installHeadWayApp = function() {
+  if (!ENV.enableHeadwayApp) {
+    // eslint-disable-next-line no-console
+    return console.log("Head Way App Disable");
+  }
+  // window.$crisp = [];
```
Comment (RaiVaibhav) [inline app/utils/trigger-headwayapp.js]: Needs to be remove

**Comment by subho007** [comment]
@RaiVaibhav please rebase this. @Praseetha-KR please merge this today 👍

**Comment by guganabu** [comment]
@subho007 @Praseetha-KR @cosmosgenius need your attention!

**Comment by guganabu** [comment]
The integration will be done in the later release...

**Pull request closed without merging**